src/test/compile-fail/regions-escape-loop-via-vec.rs

```diff
@@ -12,10 +12,18 @@
 fn broken() {
     let mut x = 3;
     let mut _y = vec!(&mut x);
+    //~^ NOTE reference must be valid for the block suffix following statement 1 at 14:30...
+    //~| NOTE borrow of `x` occurs here
+    //~| NOTE borrow of `x` occurs here
+    //~| NOTE borrow of `x` occurs here
```

Comment:
Oh ok. I noticed that this test is failing because someone else fixed
another error before I pulled. I'll fix that up tonight or tomorrow
morning.

On Fri, Aug 19, 2016, 7:01 PM Jonathan Turner <notifications@github.com>
wrote:

> In src/test/compile-fail/regions-escape-loop-via-vec.rs
> <https://github.com/rust-lang/rust/pull/35817#discussion_r75563067>:
>
> > @@ -12,10 +12,18 @@
> >  fn broken() {
> >      let mut x = 3;
> >      let mut _y = vec!(&mut x);
> > +    //~^ NOTE reference must be valid for the block suffix following statement 1 at 14:30...
> > +    //~| NOTE borrow of `x` occurs here
> > +    //~| NOTE borrow of `x` occurs here
> > +    //~| NOTE borrow of `x` occurs here
>
> Yup, that's right. It actually shows up that many times in the test since
> there are multiple tests that will label that same span.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/rust-lang/rust/pull/35817/files/005a8909800597ed37163c1019ba54c5e2520466#r75563067>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAyu2OzVtpqwrKFnqfHD2wQZQo4E0_DDks5qhjXYgaJpZM4JoFfF>
> .
>